catalog-info.yaml

```diff
@@ -436,11 +436,15 @@ spec:
       repository: elastic/logstash
       pipeline_file: ".buildkite/exhaustive_tests_pipeline.yml"
       provider_settings:
-        trigger_mode: none
-      cancel_intermediate_builds: true
-      skip_intermediate_builds: true
+        build_branches: true
+        build_pull_request_forks: false
+        build_pull_requests: false
+        build_tags: false
+        trigger_mode: code
+      cancel_intermediate_builds: false
+      skip_intermediate_builds: false
```

Comment:
I'd like to discuss what we want to do here; the above two lines `cancel_intermediate_builds` / `skip_intermediate_builds`, when set to `true` cancel currently running jobs (per branch) when a new commit is pushed.

This makes sense e.g. for PRs, but for intake style pipelines like this one, I think we want to let each job run completely, even if a subsequent commit lands soon after.

We can also tune this behavior per branch; e.g. [elastic-agent](https://github.com/elastic/elastic-agent/blob/main/catalog-info.yaml#L51) sets to `true` by default, but only for non `main` branches:

```
      cancel_intermediate_builds: true
      cancel_intermediate_builds_branch_filter: "!main"
      skip_intermediate_builds: true
      skip_intermediate_builds_branch_filter: "!main"
```

Reply:
it's fine that we don't cancel or skip for non-PRs, rarely we have a lot of commits landing on a given branch, so we can afford to have each commit properly/individually tested.

Reply:
Ok I'll keep things as they are then, thanks.